packages/react-card/Spec.md

```diff
@@ -0,0 +1,432 @@
+  <CardBody>
+    Donut chocolate bar oat cake. Dragée tiramisu lollipop bear claw. Marshmallow pastry jujubes toffee sugar plum.
+  </CardBody>
+</Card>
+```
+
+### Task
+
+```jsx
+<Card>
+  <CardHeader header="Alert in Teams when a new document is uploaded in channel" description="By Microsoft" />
+  <CardFooter>
+    <span>Automated</span>
+    <span>3290 uses</span>
+  </CardFooter>
+</Card>
+```
+
+### Image
+
+```jsx
+<Card>
+  <CardPreview src="monkey.png" alt="image of monkey" />
+  <CardBody>
+    <div>
+      <span>Length</span>
+      <span>4.6-39.6 IN</span>
+      <span>11.7-100.6 cm</span>
+    </div>
+    <div>
+      <span>Diet</span>
+      <span>Omnivore</span>
+      <span>nuts, fruits, eggs</span>
+    </div>
+  </CardBody>
+  <CardFooter>
+    <Button>Share</Button>
+  </CardFooter>
+</Card>
+```
+
+## Behaviours
+
+### Mouse
+
+### Interactive cards
+
+![Visual interaction of interactive cards using the mouse](./assets/interactive-interaction-mouse.png)
+
+The card surface can be used as a clickable target
+
+1. Rest
+
+2. OnHover
+
+   Cursor style changes to pointer, background color & elevation values change to show card is an actionable element
+
+3. OnClick
+
+   Executes action assigned to card i.e. “Open file, Open link in browser etc...”
+
+#### Selectable cards
+
+![Visual interaction of selectable cards using the mouse](./assets/selectable-interaction-mouse.png)
+
+Clicking the card surface can toggle the checkbox state to `selected`. Note that checkboxes are composed together as an additional component and are not provided by default.
+
+1. Rest
+
+   Default state is `unselected`
+
+2. Hover
+
+   Cursor style changes to `pointer`, background color and elevation follows interactive styling to show visual feedback
+
+3. Pressed
+
+   Cursor does not change background color and elevation follows interactive styling to show visual feedback for pressed state.
+
+4. onClick
+
+   Card and Checkbox are both in selected state, clicking again toggles components to unselected state
+
+### Non-interactive cards
+
+![Visual interaction of non-interactive cards using the mouse](./assets/non-interactive-interaction-mouse.png)
+
+Cards that have multiple actionable child components are expected to not be clickable. We preserve the use of `EnterKey` for moving focus to actionable controls inside the card, therefore the card surface should not be used to set the state of the card but is reliant on the checkbox or similar inputs to set the card state.
+
+If users have a scenario that requires them to use actionable components within a clickable Card then the bubbling of events is expected to be managed by the users themselves.
+
+Mouse interaction
+
+- OnMouseEnter / Hover
+
+  Cursor style stays at default, rest state stays persistent no visual feedback is applied to the card surface
+
+- Hover & onClick (Checkbox)
+
+  Hover state & pressed state applies to checkbox component
+
+- onClick
```

Comment:
Does this `onClick` apply to the entire card ? wouldn't that just turn the card interactive ?